actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -91,10 +91,12 @@ class ActionCable.Connection
           @subscriptions.reload()
         when message_types.ping
           @monitor.recordPing()
-        when message_types.confirmation
+        when message_types.confirm_subscribe
           @subscriptions.notify(identifier, "connected")
         when message_types.rejection
           @subscriptions.reject(identifier)
+        when message_types.confirm_unsubscribe
+          @subscriptions.notify(identifier, "unsubscribed")
```

Comment:
Should call this `disconnected` to match `connected` when the subscription is confirmed.